src/server.rs

```diff
@@ -14,6 +14,7 @@ pub async fn run_server(settings: ServerSettings, tx: mpsc::Sender<model::Report
     Server::bind(&settings.host.parse::<SocketAddr>()?)
         .serve(
             Router::new()
+                .route("/health", get(health_check))
```

Comment:
I didn't find out, why only this new line is detected as untested, while the other are not. The code is anyways tested in https://github.com/famedly/barad-dur/blob/25d36c2aeb2f7ae7a0df6d50e8fef011e56f13a0/src/tests/mod.rs#L72
I think, this can be ignored for now.